apps/mobile/src/navigation/linking/getStateFromPath.test.js

```diff
@@ -0,0 +1,150 @@
+import getStateFromPath from 'navigation/linking/getStateFromPath'
+
+// // Transcription of "notifications paths" from Web
+// '/my/account',
```

Comment:
Was this the only /my/ path you found or just testing only this one?

Reply:
Yeah, again the actual test cases in here are, other than the ones transcribe from the spreadsheet relatively arbitrary. Add things or take things out :) If Tibet gets us a master list we can just put them all in here.